docs/whats_new/version_ongoing.rst

```diff
@@ -95,3 +95,10 @@ In addition, if the tp_layer and uds_layer parameters are not given at yaml leve
 a default configuration is applied.
 
 See :ref:`uds_auxiliary`
+
+Kiso log levels
+^^^^^^^^^^^^^^^
+To let users decide the level of information they want to see in their logs, new log levels
+have been defined. When launch normally only the logs in the tests and the errors will be
+active.
+The flag --verbose should be used to display the internal logs of the framework.
```

Comment:
Add those info also to the general docs. Maybe  leave just a link here

Reply:
Done